crates/ironrdp-cliprdr/src/pdu/format_list.rs

```diff
@@ -6,11 +6,164 @@ use ironrdp_pdu::{cast_int, ensure_size, invalid_message_err, PduDecode, PduEnco
+
+    /// A memory object containing a BITMAPINFO structure followed by the bitmap bits.
+    pub const CF_DIB: Self = Self(8);
+
+    /// Handle to a color palette. Whenever an application places data in the clipboard that
+    /// depends on or assumes a color palette, it should place the palette on the clipboard as well.
+    /// If the clipboard contains data in the CF_PALETTE (logical color palette) format, the
+    /// application should use the SelectPalette and RealizePalette functions to realize (compare)
+    /// any other data in the clipboard against that logical palette. When displaying clipboard
+    /// data, the clipboard always uses as its current palette any object on the clipboard that is
+    /// in the CF_PALETTE format.
+    ///
+    /// NOTE: When transfered over `CLIPRDR`, [`crate::pdu::format_data::ClipboardPalette`] structure
+    /// is used instead of `HPALETTE`.
+    pub const CF_PALETTE: Self = Self(9);
+
+    /// Data for the pen extensions to the Microsoft Windows for Pen Computing.
+    pub const CF_PENDATA: Self = Self(10);
+
+    /// Represents audio data more complex than can be represented in a CF_WAVE standard wave format.
+    pub const CF_RIFF: Self = Self(11);
+
+    /// Represents audio data in one of the standard wave formats, such as 11 kHz or 22 kHz PCM.
+    pub const CF_WAVE: Self = Self(12);
+
+    /// Unicode text format. Each line ends with a carriage return/linefeed (CR-LF) combination.
+    /// A null character signals the end of the data.
+    pub const CF_UNICODETEXT: Self = Self(13);
+
+    /// A handle to an enhanced metafile (HENHMETAFILE).
+    ///
+    /// NOTE: When transfered over `CLIPRDR`, [`crate::pdu::format_data::PackedMetafile`] structure
+    /// is used instead of `HENHMETAFILE`.
+    pub const CF_ENHMETAFILE: Self = Self(14);
+
+    /// A handle to type HDROP that identifies a list of files. An application can retrieve
+    /// information about the files by passing the handle to the DragQueryFile function.
+    pub const CF_HDROP: Self = Self(15);
+
+    /// The data is a handle (HGLOBAL) to the locale identifier (LCID) associated with text in the
+    /// clipboard. When you close the clipboard, if it contains CF_TEXT data but no CF_LOCALE data,
+    /// the system automatically sets the CF_LOCALE format to the current input language. You can
+    /// use the CF_LOCALE format to associate a different locale with the clipboard text. An
+    /// application that pastes text from the clipboard can retrieve this format to determine which
+    /// character set was used to generate the text. Note that the clipboard does not support plain
+    /// text in multiple character sets. To achieve this, use a formatted text data type such as
+    /// RTF instead.The system uses the code page associated with CF_LOCALE to implicitly convert
+    /// from CF_TEXT to CF_UNICODETEXT. Therefore, the correct code page table is used for the
+    /// conversion.
```

Comment:
Keep in mind that the first few lines must be a summary, and this summary is separated from the rest by a newline:

```suggestion
    /// The data is a handle (HGLOBAL) to the locale identifier (LCID) associated with text in the
    /// clipboard.
    ///
    /// When you close the clipboard, if it contains CF_TEXT data but no CF_LOCALE data,
    /// the system automatically sets the CF_LOCALE format to the current input language. You can
    /// use the CF_LOCALE format to associate a different locale with the clipboard text. An
    /// application that pastes text from the clipboard can retrieve this format to determine which
    /// character set was used to generate the text. Note that the clipboard does not support plain
    /// text in multiple character sets. To achieve this, use a formatted text data type such as
    /// RTF instead.The system uses the code page associated with CF_LOCALE to implicitly convert
    /// from CF_TEXT to CF_UNICODETEXT. Therefore, the correct code page table is used for the
    /// conversion.
```

You can check how it’s rendered using `cargo doc --open`.

See [rustdoc book](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components) for reference.

I think there are other places that need to be addressed in a similar fashion.

Reply:
done